Import-BitBucket_exclude.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$Path = "C:\Dropbox\athenaGIT\BitbucketPS"
```

Comment:
$Path should be generic and the rest should be in the loading of the module (`.psm1`)

Reply:
Again, shouldn't have made it into the repo, not sure why gitignore ignored it (see what I did there)